frontend/src/components/Review/Review.tsx

```diff
@@ -55,50 +57,73 @@ const ReviewComponent = ({ review }: Props): ReactElement => {
   return (
     <Grid item>
       <Card className={root} variant="outlined">
-        <CardContent>
-          <Grid container spacing={2}>
-            <Grid item container justify="space-between">
-              <Grid container spacing={2}>
+        <Box minHeight="200px">
+          <CardContent>
+            <Grid container spacing={2}>
+              <Grid item container justify="space-between">
+                {overallRating !== null ? (
```

Comment:
There's one review in the db that technically has all ratings of 0, not null, so it's still showing up. I highlighted it here in the console. But this probably isn't something you have to change in this PR because I think it's more of an issue of users being able to submit unrated reviews right now, and that's mentioned in [issue #63 ](https://github.com/cornell-dti/cu-apts/issues/63)

<img width="1295" alt="Screen Shot 2021-05-06 at 4 42 17 PM" src="https://user-images.githubusercontent.com/57203639/117363427-939a1280-ae8a-11eb-8654-354aef29c3e5.png">

Reply:
Yeah, null ratings are for ITU reviews. We shouldn't be allowing users to post ratings of 0, like you said.